packages/access-api/package.json

```diff
@@ -55,14 +58,14 @@
     "git-rev-sync": "^3.0.2",
     "hd-scripts": "^4.0.0",
     "is-subset": "^0.1.1",
-    "miniflare": "^2.11.0",
+    "miniflare": "2.12.1",
```

Comment:
I will try reverting back to `^` range now that I worked out conflicts across versions